### PR TITLE
CMake: Remove old references to the "Feta" codename for NVC++.

### PR DESCRIPTION
**Pull request by brycelelbach**

### CODE REVIEW

**Comment by brycelelbach** [comment]
ok to test

File: CMakeLists.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@ if (NOT THRUST_TOPLEVEL_PROJECT)
   return()
 endif()
 
-option(THRUST_ENABLE_HEADER_TESTING "Test that all public headers compile." "ON")
+option(THRUST_ENABLE_HEADER_TESTING "Test that all public headers compile." ON)
```
Comment (allisonvacanti) [inline CMakeLists.txt:46]: Both forms are valid, but why change one and not the others?

**Review by allisonvacanti — APPROVED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (brycelelbach) [reply CMakeLists.txt:46]: I don't think I did this intentionally, it was just something I copied pasted.

Comment (brycelelbach) [reply CMakeLists.txt]: Fixed.

**Pull request merged**